package.json

```diff
@@ -60,7 +60,9 @@
 		"@salesforce/ts-sinon": "^1.1.2",
 		"@types/chai": "^4",
 		"@types/cross-spawn": "^6.0.2",
+		"@types/eslint": "^8.4.1",
 		"@types/html-escaper": "^3.0.0",
+		"@types/json-schema": "^7.0.11",
```

Comment:
`@types/eslint` uses this package to define some of its own types, and adding it ourselves lets us use it for matching definitions.
3PP pending manual approval for some reason: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE0000002RH3YAM/view